draft-ietf-spice-sd-cwt.md

```diff
@@ -490,7 +490,7 @@ sd-cwt-issued = #6.18([
 ])
 
 sd-protected = {
-   &(typ: 16) ^ => "application/sd+cwt",
+   &(typ: 16) ^ => "application/sd+cwt" / uint, ; TODO: IANA request
```

Comment:
You should add the IANA request in the PR, so we don't need to merge a TODO into the draft.